### ISSUE TITLE
failed to find address of CRenderer

### ISSUE THREAD

**Issue by kidmaomao** [cursey/kanan-new#285]
![QQ图片20230517202526](https://github.com/cursey/kanan-new/assets/31175688/b38de453-8652-4463-93f1-4bbb133e1819)

after game version update (talent renovation S4), poping out this Error note, and client just crash

**Comment by yy10112002** [comment]
Now US server have the same issue

**Comment by Nos-mirc** [comment]
Experiencing the same thing, apparently this is something that the other regions went through when they release the Renovation S4 update.

**Comment by Hallowen12** [comment]
^ Any update or resolution would be much appreciated.

**Comment by Kothair** [comment]
yea i seem to be also having this problem

**Comment by kimyoungwoong12** [comment]
Everyone, I have this problem, too.

**Comment by Nos-mirc** [comment]
Go check out the newest version from Actions.

**Comment by kimyoungwoong12** [comment]
> Go check out the newest version from Actions at the top. It's unapproved atm but it's working.

Wow, it works really well Thank you.

**Comment by Ittorasi** [comment]
After dl'ing new files mabi just instantly closes after trying to load kanan, friend of mine got the same issue, cant get the game to load past the white screen. something i should do different like full re-install?

**Comment by Nos-mirc** [comment]
> > After dl'ing new files mabi just instantly closes after trying to load kanan, friend of mine got the same issue, cant get the game to load past the white screen. something i should do different like full re-install?
> 
> This has been an issue for me as well, I've done a full re-install and fresh kanan files.

Yeah do this but also you can save and copy the Profiles DAT file so you don't have to re-input all your logins.
